bindings/kafka/0.4.0/channel.json

```diff
@@ -6,7 +6,7 @@
   "type": "object",
   "additionalProperties": false,
   "patternProperties": {
-    "^x-[\\w\\d\\.\\-\\_]+$": {
+    "^x-[\\w\\d\\.\\x2d_]+$": {
```

Comment:
Sorry, I'm struggling to parse this - what is the aim of this change?

Reply:
If I recall correctly it was related to making the pattern property `unicode ECMA-262 compliant` cc @smoya 

It was introduced here: https://github.com/asyncapi/spec-json-schemas/pull/145

This is the regex format JSON Schema dictates: https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-validation-01#section-7.3.8

Reply:
This is what all spec and binding schema files use btw 🙂